servicetalk-utils-internal/src/main/java/io/servicetalk/utils/internal/PlatformDependent0.java

```diff
@@ -142,37 +146,56 @@ final class PlatformDependent0 {
                         MethodType.methodType(void.class, Object.class),
                         throwExceptionMH,
                         MethodType.methodType(void.class, Throwable.class));
-                throwConsumer = (Consumer<Throwable>) throwExceptionCallSite.getTarget().bindTo(unsafe).invoke();
-            } catch (Throwable all) {
-                LOGGER.warn("failed computing THROW_EXCEPTION", all);
+                //noinspection unchecked
+                Consumer<Throwable> unsafeThrowConsumer =
+                        (Consumer<Throwable>) throwExceptionCallSite.getTarget().bindTo(unsafe).invoke();
+                throwConsumer = (t) -> {
+                    try {
+                        // JVM has been observed to crash when passing a null argument.
+                        // See https://github.com/netty/netty/issues/4131.
+                        unsafeThrowConsumer.accept(null != t ? t : new NullPointerException("Throwable was null"));
```

Comment:
Can we use `Objects.requireNonNull` here?

Reply:
The intention was to have `unsafeThrowConsumer` throw rather the fall-back sneaky throws.

Reply:
Looks like after removing the catch block we can start using `Objects.requireNonNull`

Reply:
OK, I moved it to `throwException` so that it would be consistent on both paths.